src/main/java/org/hawkular/client/android/auth/ModifiedAuthzModule.java

```diff
@@ -0,0 +1,197 @@
+
+                    } catch (Exception e) {
+                        Log.e(ModifiedAuthzModule.class.getSimpleName(),
+                                "Error with Login", e);
+                        exception = e;
+                        return null;
+                    }
+
+                    return result;
+                }
+
+                @Override
+                protected void onPostExecute(HeaderAndBody headerAndBody) {
+                    if (exception == null) {
+                        callback.onSuccess(new String(headerAndBody.getBody()));
+                    } else {
+                        callback.onFailure(exception);
+                    }
+                }
+            }.executeOnExecutor(AsyncTask.THREAD_POOL_EXECUTOR);
+        }
+    }
+
+
+    private String buildLoginData(String username, String password) {
+        String cred = username + ":" + password;
+        return new String(Base64.encode(cred.getBytes(), Base64.NO_WRAP));
+    }
+
+    @Override
+    public boolean refreshAccess() {
+        return true;
+    }
+
+    @Override
+    public void deleteAccount() {
+        sessionStore.remove("hawkular");
+    }
+
+    @Override
+    public AuthorizationFields getAuthorizationFields(URI requestUri, String method, byte[] requestBody) {
+        AuthorizationFields fields = new AuthorizationFields();
+        OAuth2AuthzSession storedAccount = sessionStore.read("hawkular");
+        String cred = new String(Base64.encode((storedAccount.getAccessToken()).getBytes(), Base64.NO_WRAP));
+        fields.addHeader("Authorization", "Basic " + cred);
+
+        return fields;
+    }
+
+    @Override
+    public ModuleFields loadModule(URI relativeURI, String httpMethod, byte[] requestBody) {
+        AuthorizationFields authzFields = getAuthorizationFields(relativeURI, httpMethod, requestBody);
+        ModuleFields moduleFields = new ModuleFields();
+        moduleFields.setHeaders(authzFields.getHeaders());
+        moduleFields.setQueryParameters(authzFields.getQueryParameters());
+        return moduleFields;
+    }
+
+    @Override
+    public boolean handleError(HttpException exception) {
+        return false;
+    }
+
+    private SQLStore<OAuth2AuthzSession> openSessionStore() {
+        DataManager.config("sessionStore", SQLStoreConfiguration.class)
+                .forClass(OAuth2AuthzSession.class)
+                .withContext(applicationContext)
+                .withIdGenerator(new IdGenerator() {
+                    @Override
+                    public String generate() {
+                        return UUID.randomUUID().toString();
+                    }
+                }).store();
+
+        sessionStore = (SQLStore<OAuth2AuthzSession>) DataManager.getStore("sessionStore");
+        sessionStore.openSync();
+        return sessionStore;
+    }
+
+    public void addAccount(OAuth2AuthzSession session) {
+        sessionStore.save(session);
+    }
+
```

Comment:
Just have a single line between methods. As far as I know Hawkular doesn’t have any strict rules except Checstyle ones. You can take a look at [Google ones](https://google.github.io/styleguide/javaguide.html) for some motivation and inspiration.